### PR TITLE
feat: Licensing & stuff

### PR DESCRIPTION
**Pull request by JanCizmar**

### CODE REVIEW

File: webapp/src/component/common/ButtonGroupRouter.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    ?.path;
+
+  return (
+    <Button
+      component={Link}
+      to={props.link}
+      size="small"
+      disableElevation
+      color={active ? 'primary' : 'default'}
+      data-cy="billing-subscriptions-cloud-button"
```
Comment (stepan662) [inline webapp/src/component/common/ButtonGroupRouter.tsx:21]: Is this a general component? Seems like a specific for billing.

File: webapp/src/views/organizations/billing/Subscriptions/cloud/Plans/usePlan.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { T } from '@tolgee/react';
 
 import { components } from 'tg.service/billingApiSchema.generated';
 import { useBillingApiMutation } from 'tg.service/http/useQueryApi';
-import { useOrganization } from 'tg.views/organizations/useOrganization';
+import { useOrganization } from '../../../../useOrganization';
```
Comment (stepan662) [inline webapp/src/views/organizations/billing/Subscriptions/cloud/Plans/usePlan.tsx:5]: 😢

File: webapp/src/views/organizations/billing/Subscriptions/common/Plan.tsx [hunk 3]
```diff
@@ -1,26 +1,27 @@
 import { styled } from '@mui/material';
 
-export const StyledPlan = styled('div')`
+export const Plan = styled('div')`
```
Comment (stepan662) [inline webapp/src/views/organizations/billing/Subscriptions/common/Plan.tsx:3]: I thought we'll have this convention to include `Styled` prefix

Comment (JanCizmar) [reply webapp/src/views/organizations/billing/Subscriptions/common/Plan.tsx:3]: I just don't feel, if it's good idea for exported common components. The user doesn't care if it's "real" component or styled component, right?

**Review by stepan662 — APPROVED** [review]

**Pull request merged**